packages/api-db-mongodb/src/migration.ts

```diff
@@ -789,6 +797,73 @@ export const Migrations: Migration[] = [
       await images.updateMany({}, {$rename: {source: 'link'}})
       await images.updateMany({}, {$rename: {author: 'source'}})
     }
+  },
+  {
+    // change embed block properties width and height from number to string
+    version: 20,
+    async migrate(db) {
+      const articles = db.collection(CollectionName.Articles)
+      const migrationArticles = await articles.find().toArray()
+
+      for (const article of migrationArticles) {
+        if (article.draft) {
+          article.draft.blocks.forEach((block: ArticleBlock) => {
```

Comment:
Why not use, you can prevent code duplication like this
```
for (const key in article) {
    const blocks = article[key].block
   ....
}
```
Or are there articles of a state you dont want to change?